package.json

```diff
@@ -82,6 +82,7 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "rimraf": "^3.0.1",
+    "shx": "^0.3.2",
```

Comment:
I'm afraid that adding `shx` is acceptable or not because `shx` provides various unix like commands in addition to `mv`.
Please let me know If you have any recommendation about platform-independent `mv` :bow: